3p/3d-gltf/viewer.js

```diff
@@ -58,8 +58,7 @@ export default class GltfViewer {
     this.ampPlay_ = true;
 
     /** @private */
-    this.ampInViewport_ =
-      options['initialIntersection']['intersectionRatio'] > 0;
+    this.ampInViewport_ = false;
```

Comment:
Could you explain this change?

Reply:
Sure. My impression is that `3d-gltf/viewer.js` is using the `initialIntersection` to set the `ampInViewport` variable the first time.

I was thinking it is likely fine to be initially incorrect within the iframe, since it'll get set very quickly by `viewportCallback` (which postMessages viewport status).

Reply:
Probably ok, yes. But still wondering why remove it. Do we not supply this option anymore? I thought we continued it?

Reply:
I only continued it for the few ads components that use it as opposed to all clients of `getIframe`

Reply:
sounds good.